pgjdbc/src/main/java/com/yugabyte/ysql/YBManagedHostnameVerifier.java

```diff
@@ -1,16 +1,14 @@
 package com.yugabyte.ysql;
 
+import static org.postgresql.util.internal.Nullness.castNonNull;
```

Comment:
Is the change in order of import deliberate?

Reply:
This and other similar changes were done by a gradle task (I think it's `checkstyleMain`) while building/packaging the project.